pkg/serverless/invocationlifecycle/trace_test.go

```diff
@@ -7,27 +7,48 @@ package invocationlifecycle
 
 import (
 	"os"
+	"strconv"
 	"testing"
 	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/trace/api"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestStartExecutionSpan(t *testing.T) {
-	startTime := time.Now()
-	startExecutionSpan(startTime)
+var testString = `a5a{"resource":"/users/create","path":"/users/create","httpMethod":"GET","headers":{"Accept":"*/*","Accept-Encoding":"gzip","x-datadog-parent-id":"1480558859903409531","x-datadog-sampling-priority":"1","x-datadog-trace-id":"5736943178450432258"}}0`
+var invalidTestString = `a5a{"resource":"/users/create","path":"/users/create","httpMethod":"GET","headers":{"Accept":"*/*","Accept-Encoding":"gzip","x-datadog-parent-id":"abcdef","x-datadog-sampling-priority":"1","x-datadog-trace-id":"abcdef"}}0`
```

Comment:
Nit: For the IDs, instead of using `"abcdef"`, I would use `"INVALID"`. They are both equally invalid, but this would make it easy to see exactly what part of the string is invalid.